docs/sources/alerting/configure-notifications/manage-contact-points/integrations/configure-mqtt.md

```diff
@@ -0,0 +1,135 @@
+---
+canonical: https://grafana.com/docs/grafana/latest/alerting/configure-notifications/manage-contact-points/integrations/configure-mqtt/
+description: Configure the MQTT notifier integration for Alerting
+keywords:
+  - grafana
+  - alerting
+  - guide
+  - contact point
+  - mqtt
+labels:
+  products:
+    - cloud
+    - enterprise
+    - oss
+menuTitle: MQTT notifier
+title: Configure the MQTT notifier for Alerting
+weight: 80
+---
+
+# Configure the MQTT notifier for Alerting
+
+The MQTT notifier integration allows you to send alert notifications to an MQTT broker.
+
+There are two formats for the MQTT notifier integration:
+
+- **JSON**: Sends the alert notification in JSON format.
+- **Text**: Sends the rendered alert notification message in a plain text format.
+
+## MQTT JSON payload
```

Comment:
could you add steps for this? what does the user do with the MQTT JSON payload? We introduce procedures with "To configure the MQTT JSON payload for Alerting, complete the following steps."

Reply:
Sure, I added a section on how to configure the integration, following the example of other integrations. Please have a look: https://github.com/grafana/grafana/pull/91487/commits/5029559c4605e8d2796ca3cee8aadd7157626e1b